comms/email/mockemail/mockemail.go

```diff
@@ -16,11 +16,12 @@ func NewClient() *Client {
 	}
 }
 
-func (c *Client) Send(to []string, message json.RawMessage, tplID email.TplID) error {
+func (c *Client) Send(to []string, cc []string, message json.RawMessage, tplID email.TplID) error {
```

Comment:
While I can understand making `to []string`, I can't find anywhere in the code that passes more than a single value. Could we simplify this and make this just take `to string` (and `cc string`) to match the API interface?

If not, why not extend the API to be general as well and make `AddPolicyCoverageReq` (and others) take `to []string` and `cc []string`? As-is, this seems to expose a much less generic API than what is implemented under the hood. Why have this additional complexity internally if we are not exposing it to the user?

Reply:
These are great points you brought up @ephjos . Our general approach tends to be to limit the public API contracts to only what is needed, but to keep the internal components more open. The thinking is that because the underlying `sendgrid/mail/mail_v3` library supports multiple recipients that we want to support them internally for any other callers that need the functionality. The `mockemail` package is only meant to reproduce the `sendgrid` package functionality for internal testing. But yeah... restricting them all down internally would solve the current "problem".

But lets follow your idea about expanding the public api contracts to support multiples. I think this is a good idea because that way even if we only need singles right now we could move to support multiples in the future without breaking the public data contract. Would you mind following that up, and then see how things play out in the unit tests? For the purposes of the challenge just update the new new coverage endpoint to support multiples of both to and cc, don't worry about the others. You'll need to get the unit test to pass with a minimum of 2x to and 2x cc fields to fully flex it.

Reply:
Thanks, I'll take a look at expanding that contract and get something pushed up!